docs/types.rst

```diff
@@ -617,6 +617,33 @@ Because of this, mappings do not have a length or a concept of a key or value be
 Mappings are only allowed for state variables (or as storage reference types
 in internal functions).
 
+It is possible to mark mappings ``public`` and have Solidity create an accessor.
+The ``_KeyType`` will become a required parameter for the accessor and it will
```

Comment:
If the value type itself is only a mapping or an array, it will recursively add another parameter to the accessor function.